github-orgs/kubeflow/org.yaml

```diff
@@ -962,6 +962,17 @@ orgs:
             - Bobgy
             - paveldournov
             - theadactyl
+          release-managers:
```

Comment:
Why not re-using the existing release-team?

Reply:
We want to limit write access to those with lead roles of the release team: release manager, release docs lead, and release product manager.

Therefore, we have a PR to remove write access from the existing release-team https://github.com/kubeflow/internal-acls/pull/511 but contains all release team members and this PR to create a separate GitHub team for the leads with write access.